src/modules/ContentList/test.tsx

```diff
@@ -105,17 +119,33 @@ describe('ContentList', () => {
     });
 
     test('handles selecting content', async () => {
-        const { rendered } = renderWithProvider(<ContentList />, {
-            executeQuery: () =>
-                fromValue({
-                    data: {
-                        content: contentList,
-                    },
-                }),
-        });
+        await actAndFlush(() =>
+            renderWithProvider(<ContentList />, {
+                executeQuery: ({ query }: { query: DocumentNode }) => {
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
These comments/ignore can actually go now that we have this utility to retrieve the name.